diamm/helpers/object_pagination.py

```diff
@@ -0,0 +1,30 @@
+from collections import OrderedDict
+import re
+
+from rest_framework.response import Response
+from rest_framework.pagination import PageNumberPagination
+
+class Object_Pagination(PageNumberPagination):
```

Comment:
It automatically paginates but I assume we also want a list of numbers to navigate those pages (like we do in search results.) The built-in paginators [don't respond with a list of all page urls](https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/pagination.py#L224)